### PR TITLE
SPARKC-244: Docs for writing TTL and WRITETIME

### PR DESCRIPTION
**Pull request by jacek-lewandowski**

### CODE REVIEW

File: doc/5_saving.md [hunk 1]
```diff
@@ -97,6 +97,70 @@ val company = Company("DataStax", address)
...
+values for them have to be used, there are a couple options provided by the API. 
+
+TTL and WRITETIME options are specified as properties of `WriteConf` object, which can be optionally 
+passed to `saveToCassandra` method. TTL and WRITETIME options are specified independently from one 
+another. 
+
+### Using a constant value for all rows
+When the same value should be used for all the rows, one can use the following syntax:
+
+```scala
```
Comment (bcantoni) [inline doc/5_saving.md]: Add "import com.datastax.spark.connector.writer._" so examples will work in Spark shell

File: doc/5_saving.md [hunk 2]
```diff
@@ -97,6 +97,70 @@ val company = Company("DataStax", address)
...
+rdd.saveToCassandra(ks, tab, writeConf = WriteConf(ttl = TTLOption.perRow("ttl")))
+rdd.saveToCassandra(ks, tab, writeConf = WriteConf(timestamp = TimestampOption.perRow("timestamp")))
+```
+
+`perRow(String)` method accepts a name of a property in each RDD item, which value will be used as TTL 
+or WRITETIME value for the row. 
+
+Say we have an RDD with `KeyValueWithTTL` objects, defined as follows:
+```scala
```
Comment (bcantoni) [inline doc/5_saving.md]: Add "import com.datastax.spark.connector.writer._" so examples will work in Spark shell

**Comment by bcantoni** [comment]
My other suggestion is to change from keyspace 'ks' to 'test' in order to match the other examples on this page.

**Comment by pkolaczk** [comment]
+1

**Pull request merged**